src/main/java/com/alkemy/ong/service/UserService.java

```diff
@@ -104,24 +104,24 @@ private String uploadImage(MultipartFile file) {
         return amazonService.uploadFile(file);
     }
 
-    public User loginUser(User user) throws UserNotFoundException, InvalidPasswordException {
+    public User loginUser(User user) throws WrongValuesException {
         if (userRepository.existsByEmail(user.getEmail())) {
             String email = user.getEmail();
             String password = user.getPassword();
             UserModel userModel = userRepository.findByEmail(email);
             return getUserPasswordChecked(password, userModel);
         } else {
-            throw new UserNotFoundException("User not found");
+            throw new WrongValuesException("User or password is incorrect");
```

Comment:
El texto de la exception como siempre va a ser el mismo, podemos ponerlo directamente en la exception así se crea siempre con ese. Además, podemos poner `User and/or password is incorrect`.